src/Utils/Hooks/UseAction.ts

```diff
@@ -0,0 +1,23 @@
+import React, { useCallback } from "react";
+import { useDispatch } from "react-redux";
+
+
+interface IProps{
+  action?:any
+}
+interface IData{
+  data?:any
+}
+
+const useAction = (action: IProps) => {
```

Comment:
The main purpose of this hook is unclear from its name. and it is not clear from the name where and when should we use it.